OnlineLibrary.BLL/Interfaces/IAuthorService.cs

```diff
@@ -2,16 +2,20 @@
 using System.Collections.Generic;
 using System.Text;
 
-namespace BookAuditNLayerApp.BLL.Interfaces
+namespace OnlineLibrary.BLL.Interfaces
 {
     public interface IAuthorService<T> where T : class
     {
-        List<T> GetAllAuthors();
+        List<T> GetAllAuthors(IDataExportService dataExportService);
```

Comment:
I suppose _IDataExportService dataExportService_ was as an example, so please remove it